canonical-web-and-design/global-nav
# Swap from Circle CI to Github actions

## Done

Swap Circle CI to Github actions

## QA

- Github actions checks should turn green:
lint
build

- Circle CI build error happens because the Circle CI files have been removed but webhook has not been removed yet, to avoid blocking possible urgent PRs
